src/clusterfuzz/_internal/bot/fuzzers/syzkaller/runner.py

```
@@ -70,7 +83,34 @@ def get_config():
 
 def get_cover_file_path():
   """Return location of coverage file for Syzkaller."""
-  return os.path.join(get_work_dir(), 'coverfile')
+  file_path = os.path.join(get_work_dir(), 'coverfile')
+  if not os.path.isfile(file_path):
+    open(file_path, 'w+').close()
+
+  logs.log(f'Rawcover file will be stored in {file_path}')
+  return file_path
+
+
+def find_syzkaller_port(pid: int) -> int:
+  """Find localhost port where syzkaller is connected."""
+
+  for connection in psutil.net_connections():
+    if connection.pid != pid:
+      continue
+
+    try:
+      local_address = connection.laddr
+
+      if (local_address.ip == LOCAL_HOST and
+          connection.status == psutil.CONN_LISTEN):
+        logs.log(f'Syzkaller port = {local_address.port}')
+        return local_address.port
+
+    except AttributeError:
+      continue
```

why do we continue in this case? This could hide real errors.